packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -25,6 +25,11 @@ const BoxConstraints _kMiniSizeConstraints = BoxConstraints.tightFor(
   height: 40.0,
 );
 
+const BoxConstraints _kLargeSizeConstraints = BoxConstraints.tightFor(
+  width: 96.0,
+  height: 96.0,
+);
+
 const BoxConstraints _kExtendedSizeConstraints = BoxConstraints(
```

Comment:
I think this is the same as `BoxConstraints.tightFor(height: 48)`

Reply:
Done